classifiers/llm/deberta_review_classifier/config.py

```diff
@@ -0,0 +1,46 @@
+from util.configs import build_classifier_premium_config
+from util.enums import State, SelectionType, BricksVariableType, RefineryDataType
+from . import deberta_review_classifier, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_classifier_premium_config(
+        function=deberta_review_classifier,
+        input_example=INPUT_EXAMPLE,
+        issue_id=302,
+        tabler_icon="MoodNerd",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        gdpr_compliant="false",
+        type="premium",
+        kern_token_proxy_usable="false",
+        docker_image="none",
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "llm",
+            "not_gdpr_compliant",
+        ],  # first entry should be parent directory
+        # bricks integrator information
+        integrator_inputs={
+            "name": "deberta_review_classifier",
+            "refineryDataType": RefineryDataType.TEXT.value,
```

Comment:
i think we need an outputs part since it's a classifier
"outputs": [
                "Positive",
                "Neutral",
                "Negative",
            ]

Reply:
Added!